src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Drawable, AreaCalculable {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color) {
+        super.setColor(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
```

Comment:
There seems to be a typo in the assignment of the `secondLeg` field. You're assigning `firstLeg` to `secondLeg`, which is incorrect if the legs are supposed to have different lengths.